### PR TITLE
Updating conenector values and fixing Tactics

### PR DESCRIPTION
**Pull request by ashwin-patil**
Fixes #

## Proposed Changes

  - Updating connector values to match with other ASIM files in same folder
  -  updating Tactics to standard values to align with others

### CODE REVIEW

**Review by shainw — COMMENTED** [review]
Approved from perspective of Tactic updates, lets get review from ASIM owners on changes to connectors.

File: Hunting Queries/ASimProcess/imProcess_ExchangePowerShellSnapin.yaml [hunk 1]
```diff
@@ -5,17 +5,20 @@ description: |
   Whilst this is a legitimate administrative tool it is abused by attackers to performs actions on a compromised
   Exchange server. Hunt for unusual activity related to this Snapin including it being added on new hosts or by
   new accounts.'
-requiredDataConnectors: []
+requiredDataConnectors:
```
Comment (oshezaf) [inline Hunting Queries/ASimProcess/imProcess_ExchangePowerShellSnapin.yaml:8]: When only ASIM sources are involved, at this time, the data connectors list should be empty. 

To explain: 
- "required" is actually always only a recommendation. No actual enforcement. It is mostly for the UX.
- In this case, imProcessCreate supports MDE, Sysmon and potentially others (WindowsEvents coming soon), so SecurityEvent is not the right value to show. We (ab)used the detection name to specify the require source, until a better solution is available.

File: Hunting Queries/ASimProcess/imProcess_HostExportingMailboxAndRemovingExport.yaml [hunk 2]
```diff
@@ -16,19 +19,19 @@ tags:
   - NOBELIUM
 query: |
 
-     // Adjust the timeframe to change the window events need to occur within to alert
```
Comment (oshezaf) [inline Hunting Queries/ASimProcess/imProcess_HostExportingMailboxAndRemovingExport.yaml:19]: @YaronFruchtmann : note that the team prefers to maintain uniformity around indents. I am with them on that :-). So let's try.

**Review by oshezaf — CHANGES_REQUESTED** [review]

**Comment by shainw** [comment]
@ashwin-patil - let's get an update on this.

File: Hunting Queries/ASimProcess/imProcess_ExchangePowerShellSnapin.yaml (hunk 1 above)
Comment (ashwin-patil) [reply Hunting Queries/ASimProcess/imProcess_ExchangePowerShellSnapin.yaml:8]: reverted connector changes to original

**Comment by ashwin-patil** [comment]
> @ashwin-patil - let's get an update on this.

Updated. resolved conflicts, reverting ASIM connector changes

**Review by yaronMSFT — APPROVED** [review]

**Review by shainw — APPROVED** [review]

**Pull request merged**